common/tools/eslint-plugin-azure-sdk/package.json

```diff
@@ -115,15 +115,15 @@
     "@typescript-eslint/parser": "~8.26.0",
     "@typescript-eslint/rule-tester": "~8.26.0",
     "@typescript-eslint/utils": "~8.26.0",
-    "@vitest/browser": "^3.0.9",
-    "@vitest/coverage-istanbul": "^3.0.9",
+    "@vitest/browser": "^3.0.3",
```

Comment:
it's downgraded probably due to merge. but both should still be resolved to 3.0.9 or later.